numpy/core/tests/examples/cython/setup.py

```diff
@@ -9,7 +9,11 @@
 from setuptools.extension import Extension
 import os
 
-macros = [("NPY_NO_DEPRECATED_API", 0)]
+macros = [
+    ("NPY_NO_DEPRECATED_API", 0),
+    # Require 1.25+ to test datetime additions
+    ("NPY_TARGET_VERSION", "NPY_1_25_API_VERSION"),
```

Comment:
Have to add this, since NumPy now defaults to not exporting these new symbols otherwise (leading to compilation warnings and runtime error if you use the new functions).